frontend/components/panels/WithdrawDAIPanel.vue

```diff
@@ -74,6 +74,10 @@ export default Vue.extend({
             type: Boolean,
             default: false,
         },
+        isRefreshing: {
```

Comment:
Do we want to rename this to just `disabled`? We currently only provide named actions when it actually changes something specific to this panel (Like a loading icon, texts etc). 

Changing it to disabled just makes it extendable if we ever need to disable it for another reason.